api/src/main/java/org/openmrs/module/esaudereports/reporting/utils/EmrReportingUtils.java

```diff
@@ -25,6 +25,7 @@ public static CohortIndicator cohortIndicator(String name, Mapped<CohortDefiniti
 		CohortIndicator ind = new CohortIndicator(name);
 		ind.addParameter(new Parameter("startDate", "Start Date", Date.class));
 		ind.addParameter(new Parameter("endDate", "End Date", Date.class));
+		ind.addParameter(new Parameter("revisionEndDate", "Revision End Date", Date.class));
 		ind.setCohortDefinition(cohort);
```

Comment:
We also need location parameter added since they will be passed at the cohort definition level. something like 
**ind.addParameter(new Parameter("location", "Location", Location.class));**